plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3378,6 +3378,7 @@ blockly.warning.no_more_argument=Adding more arguments after the message argumen
 blockly.warning.no_item_extension_fuel=Getting the fuel power from a procedure triggered from a fuel item extension condition will crash the game due to recursive call.
 blockly.warning.state_provider_int_property_warning=Data pack validation will FAIL if the block doesn't have the given property, or if the value is out of bounds.
 blockly.warning.utility_adv_trigger=Trigger {0} is a utility advancement trigger and should not be used in combination with other triggers.
+blockly.warning.powder_snow_walkable_mobs_tag_required=The Climb on top of powder snow task requires the powder_snow_walkable_mobs entity tag
```

Comment:
`For the "Climb on top of powder snow "task to work, this entity needs to be tagged under requires "minecraft:powder_snow_walkable_mobs" entity tag`

would be more fitting